recipes/inspector/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "1.0.1" %}
+
+package:
+  name: inspector
+  version: {{ version }}
+
+source:
+  url: https://github.com/ChongLab/Inspector/archive/refs/tags/inspector-v{{ version }}.tar.gz
+  sha256: fac61240ee7eff2ba0624491c6e5e050567564695cc79096f54d4e1bb66b8b35
+
+build:
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - python <3
+  run:
+    - python <3
+    - minimap2 =2.15
```

Comment:
Are those strict pins really necessary? Please consider removing the pins.

Reply:
Strict to match python 2.7. Will try to remove them later. Thank you.

Reply:
@bgruening  We have updated the codes and it supports Python 3 now. Could you help review and merge? Thank you so much in advance.